.travis.yml

```diff
@@ -56,13 +68,25 @@ matrix:
       env: TOXENV=py38-djangomaster-redislatest
     - python: 3.8
       env: TOXENV=py38-djangomaster-redismaster
+    - python: 3.9-dev
+      env: TOXENV=py39-dev-djangomaster-redislatest
+    - python: 3.9-dev
+      env: TOXENV=py39-dev-djangomaster-redismaster
   allow_failures:
     - env: TOXENV=py36-djangomaster-redislatest
     - env: TOXENV=py36-djangomaster-redismaster
     - env: TOXENV=py37-djangomaster-redislatest
     - env: TOXENV=py37-djangomaster-redismaster
     - env: TOXENV=py38-djangomaster-redislatest
     - env: TOXENV=py38-djangomaster-redismaster
+    - env: TOXENV=py39-dev-django22-redislatest
+    - env: TOXENV=py39-dev-django22-redismaster
+    - env: TOXENV=py39-dev-django30-redislatest
+    - env: TOXENV=py39-dev-django30-redismaster
+    - env: TOXENV=py39-dev-django31-redislatest
+    - env: TOXENV=py39-dev-django31-redismaster
```

Comment:
Thanks for keeping at it!

```suggestion
```

IMO, only `djangomaster` should allow failures, as it is unstable and changing. Python 3.9 is stable and no new PRs should break it.